screens/Home/MyVcs/IdInputModal.tsx

```diff
@@ -80,6 +89,21 @@ export const IdInputModal: React.FC<IdInputModalProps> = props => {
                   selectionColor={Theme.Colors.Cursor}
                   value={controller.id}
                   keyboardType="number-pad"
+                  rightIcon={
+                    <CustomTooltip
+                      title={t('toolTipTitle', {idType: controller.idType})}
+                      description={t(`toolTip${controller.idType}Description`)}
+                      width={Dimensions.get('screen').width * 0.85}
+                      height={Dimensions.get('screen').height * 0.18}
+                      triggerComponent={
+                        <Icon
+                          name="infocirlceo"
+                          type="antdesign"
```

Comment:
Apologies for being, can we also add testIDs for the tooltip for automation testing?

Reply:
@srikanth716 let's add testID to this tooltip view.
testID should be passed as prop. This is ui component will be used at multiple places, so testID should be contextual.